Week1(01-29-18).md

```diff
@@ -5,26 +5,27 @@ Before we get into it, I'd like to give a shout out to [@0xTim](https://github.c
 ## News
 
 ### [Vapor 3 Release Schedule Announced!](https://medium.com/@qutheory/vapor-3-release-schedule-1179bb080ca4)
-We're looking at the first tagged Beta release on February 9th, the first stable release on February 23rd and Vapor 3.0 sometime in mid March to coincide with the Swift 4.1 release. In addition Vapor 3 relies on the [Swift 4.1 beta](https://swift.org/download/#snapshots) as of this week, so go get it installed!
+We're looking at the first tagged Beta release on February 9th, the first stable release on February 23rd, and Vapor 3.0 sometime in mid March to coincide with the Swift 4.1 release. In addition, Vapor 3 relies on the [Swift 4.1 beta](https://swift.org/download/#snapshots) as of this week, so go get it installed!
 
 ## New Features
 
 ### [CHTTP Refactor](https://github.com/vapor/vapor/pull/1458)
-Performance was optimized for all HTTP requests. In addition convenience accessors like `req.headers` are now gone in favor of more explicit calls to `req.http.*`. So if you were using `req.headers`, that would now be `req.http.headers`. Calls to `print(...)` or `debugPrint(...)` passing in a `Request` or `Response`, now provide better information.
+- Performance was optimized for all HTTP requests. In addition, convenience accessors like `req.headers` are now gone in favor of more explicit calls to `req.http.*`. So if you were using `req.headers`, that would now be `req.http.headers`.
```

Comment:
This was originally just to be more explicit, but recently discovered to prevent tons of unnecessary copying.